### ISSUE TITLE
Add calendar view on counter details

### ISSUE THREAD

**Issue by rancidfrog** [albertvaka/bettercounter#18]
It would be really helpful to implement calendar monthly view on counter details so that you can have precise days instead of only how many per month.

Currently only this is available:
![](https://i.imgur.com/taIMZxq.jpg)

From uHabits - https://github.com/iSoron/uhabits
![](https://i.imgur.com/ImFIBzo.jpg)
The highlighted days allows users to know which days exactly each count took place.

It has graph, too for month week etc
![](https://i.imgur.com/SEyEJow.jpg)

Counter records the date to provide last time used values,
if these could be used to plot/show calender view it would be a great visual

**Comment by Mrnofish** [comment]
For some counting task, the Habits calendar view is great because you can see at a glance how much time has passed between events.

The example shown above is not great (no offense meant) because it is an actual habit, with a high repetition frequency and very regular, whereas I feel the calendar view used by Habits really shines for infrequent chores that do not necessarily fall on a certain day.

Things you need to remember to do, I don't know, twice a month, but that must be adjusted, based on circumstances outside your own control. For instance, things that cannot be done or must be avoided if it rains.

**Comment by session11** [comment]
I agree with the original post that a calendar view would be really helpful. If for some reason this is not possible, then just displaying the precise date on which the event occurred would also be more helpful than just the number of times an event occurred during a month. Hope to see this implemented soon.
